DOC: Change integrate tutorial to use solve_ivp instead of odeint (#9847)
scipy/scipy: doc/source/tutorial/integrate.rst

However, in this case, :math:`\mathbf{A}\left(t\right)` and its integral do not commute.

There are many optional inputs and outputs available when using `solve_ivp`
which can help tune the solver. These additional inputs and outputs
are not needed much of the time, however, and the three required input
arguments and the output solution suffice. The required inputs are the
function defining the derivative, *fprime*, the time span `[t_start, t_end]`
and the initial conditions vector, *y0*. The output to :obj:`solve_ivp` is 
a matrix where each row contains the solution vector for the given time 

> intervall. The initial conditions are therefore given in the first output row.

Reviewer: ```suggestion
function defining the derivative, *fprime*, the time span `[t_start, t_end]`
and the initial conditions vector, *y0*. :obj:`solve_ivp` outputs an object
whose *y* field is an array with consecutive solution values as columns.
The initial conditions are therefore given in the first output column.
```

Re: rows vs columns, this at least seems to be what https://docs.scipy.org/doc/scipy/reference/generated/scipy.integrate.solve_ivp.html currently suggests!

airy(sol1.t)[0]: [0.35502805 0.328952   0.12804768 0.04285786 0.01686411 0.00661331
 0.00235403 0.00095156]

The solution of `solve_ivp` with its standard parameters shows a big deviation

> to the airy function. To minimize this deviation relative and absolute

Reviewer: ```suggestion
to the airy function. To minimize this deviation, relative and absolute
```

airy(sol2.t)[0][::5]: [0.35502805 0.22162462 0.09180606 0.03590248 0.01219993 0.00364769
 0.00095156]
 
To specify used defined time points for the solution of `solve_ivp`, `solve_ivp`

> offers two possibilites that can also be used complementary. By passing the `t_eval`

Reviewer: ```suggestion
offers two possibilites that can also be used complementarily. By passing the `t_eval`
```

option to the function call `solve_ivp` returns the solutions of these time points
of `t_eval` in its output.

>>> import numpy as np
>>> t = np.linspace(t_start, t_end, 50)
>>> sol3 = solve_ivp(func, t_span, y0, t_eval=t)

If the time points of interest are however unknown, the `dense_ouput` option can be
used. Setting it to `True` returns a continuous solution and the solution at

> any arbitrary time point within the time span can be calculated.

Reviewer: ```suggestion
any arbitrary time point within the time span can be calculated via interpolation.
```

>>> sol4 = solve_ivp(func, t_span, y0, dense_ouput=True)
>>> print("sol4.sol(2)[1]: {}".format(sol4.sol(2)[1])
sol4.sol(2)[1]: 0.035042622042422505

> If the jacobian matrix of function is known it can can of passed to the `solve_ivp`

Reviewer: ```suggestion
If the jacobian matrix of function is known, it can be passed to the `solve_ivp`
```
